geomesa-kafka/geomesa-kafka-09-datastore/src/main/scala/org/locationtech/geomesa/kafka09/GeoMessage.scala

```diff
@@ -0,0 +1,189 @@
+/***********************************************************************
```

Comment:
It could, but I think we'll leave some of that refactoring to RC2.  This was a brute-force copy-and-paste to get working modules for all three Kafka versions quickly.  I agree with your suggestion.